src/sentry/api/serializers/models/group.py

```diff
@@ -932,6 +949,14 @@ def get_attrs(self, item_list, user):
             for item in item_list:
                 if self.has_dynamic_issue_counts and self.conditions:
                     attrs[item].update({"filtered_stats": filtered_stats[item.id]})
+                if self.has_inbox:
+                    # Add inbox dict - either False, or label + reason, etc. "comments", "integration"
+                    inbox_stats = self._get_inbox_stats(item_list)
```

Comment:
You probably want this call to be outside of the for loop

Reply:
You know me so well.